influxdb/line_protocol.py

```diff
@@ -59,6 +59,8 @@ def _escape_value(value):
                 "\n", "\\n"
             )
         )
+    if isinstance(value, int):
```

Comment:
perhaps `elif isinstance(value, integer_types)` since `six` has that to cover `int`s for py3 and `int`s/`long`s for py2